### PR TITLE
HWKMETRICS-252 Warning about batch size threshold in the logs

### PR DESCRIPTION
**Pull request by tsegismont**
HWKMETRICS-252 Warning about batch size threshold in the logs

Reduced the number of statements per batch.

Also, removed unused method and batch statement usage where not needed.

### CODE REVIEW

File: api/diff.txt [hunk 1]
```diff
@@ -1,55 +1,6 @@
...
->         <property name="context-root" value="hawkular-metrics-api-rest-1.1"/>
-diff -r '--exclude=target' api/metrics-api-jaxrs/.settings/org.eclipse.wst.common.project.facet.core.xml api/metrics-api-jaxrs-1.1/.settings/org.eclipse.wst.common.project.facet.core.xml
-5,6d4
-<   <installed facet="jst.web" version="3.1"/>
-<   <installed facet="jst.jaxrs" version="2.0"/>
-7a6,7
->   <installed facet="jst.jaxrs" version="1.1"/>
->   <installed facet="jst.web" version="3.1"/>
 Only in api/metrics-api-jaxrs: README.adoc
+Only in api/metrics-api-jaxrs-1.1: hawkular-metrics-api-jaxrs-1.1.iml
```
Comment (pilhuhn) [inline api/diff.txt]: Are we now putting .iml files to git?

Comment (tsegismont) [inline api/diff.txt]: -----Original Message-----

Comment (tsegismont) [inline api/diff.txt]: No. But this diff is updated by a a pre-commit hook that all Metrics developers have installed. I guess it's time to update it with an exclusion filter.

**Comment by tsegismont** [comment]
Please do not merge yet. I'm still seeing this with C* 2.2.1 when running rest-tests:
```
WARN  [SharedPool-Worker-1] 2015-09-21 14:53:46,974 NoSpamLogger.java:94 - Unlogged batch covering 2 partitions detected against table [hawkular_metrics.metrics_tags_idx]. You should use a logged batch for atomicity, or asynchronous writes for performance.
```

**Comment by tsegismont** [comment]
OK, can be reviewed and merged now

**Comment by tsegismont** [comment]
@stefannegrea @burmanm guys, John asked me to work on this before leaving for PTO. It is needed for Openshift integration. Can one of you have a look? Thanks!

**Comment by stefannegrea** [comment]
Thanks @tsegismont!

**Pull request merged**